lib/mayaUsd/render/vp2RenderDelegate/proxyRenderDelegate.cpp

```diff
@@ -783,6 +788,10 @@ bool ProxyRenderDelegate::getInstancedSelectionPath(
         return false;
     }
 
+#if UFE_PREVIEW_VERSION_NUM >= 2027 // #ifdef UFE_V2_FEATURES_AVAILABLE
+    auto ufeSel = Ufe::NamedSelection::get("MayaSelectTool");
+    ufeSel->append(si);
+#else
```

Comment:
We simply append any items to select to this named selection. Maya takes care of the replace/toggle/add/deselect.

Reply:
Same changes in all the plugins.